.github/workflows/tpl-deploy-app.yml

```diff
@@ -128,6 +128,7 @@ env:
   BACKEND_SERVER_REPLICAS: ${{ inputs.backend-server-replicas }}
   OCTANT_ENV: ${{ inputs.octant-env }}
   GLM_SENDER_NONCE: ${{ vars.GLM_SENDER_NONCE }}
+  MAINNET_PROPOSAL_CIDS: "QmSQEFD35gKxdPEmngNt1CWe3kSwiiGqBn1Z3FZvWb8mvK,Qmds9N5y2vkMuPTD6M4EBxNXnf3bjTDmzWBGnCkQGsMMGe"
```

Comment:
We agreed to start naming it `projects` instead of `proposals` in the codebase. Please, use only `project` var name in this PR

Reply:
ok, so with this one I will probably leave it as it is for now. Simply because we have PROPOSALS_CID in other places and it should be changed everywhere at once. Otherwise it will add even more confusion. We can update it later or when we work with the self-serve projects.